docs/modules/migrate/pages/rolling-restart.adoc

```diff
@@ -0,0 +1,45 @@
+= Upgrade Cluster to {enterprise-product-name}
+:description: You can use an in-place rolling restart to move each member on your Open Source cluster to an Enterprise cluster.
+
+{description}
+
+This approach ensures that you can move your cluster without interruption to the service. 
+Similar to the rolling upgrade process, a rolling restart allows you to move one member at a time.
+
+The codebase of your {open-source-product-name} cluster must match the target {enterprise-product-name} release. 
+Currently, rolling restart can only be used to update your 5.5 {open-source-product-name} cluster members.
```

Comment:
You can replace 5.5 with current version.

Going forwards, it will be possible to update each version from OS to EE.

Reply:
I have removed the line, as I think it is covered by the previous statement on matching codebases.

The issue is - if we change to current version - this could be interpreted as their OS current version, so any version, which would be incorrect.

Feel free to disagree ;-)